pkg/metrics/metrics.jsx

```diff
@@ -238,6 +238,11 @@ function make_rows(rows, rowProps, columnLabels) {
     );
 }
 
+async function get_pcp_packagelist() {
+    const os_release = await read_os_release();
+    return get_manifest_config_matchlist("metrics", "python_pcp", "python3-pcp", [os_release.PLATFORM_ID, os_release.ID]);
```

Comment:
So while I get that `pcp` depends on `python3-pcp`, the opposite is probably not true? I.e. installing python3-pcp doesn't guarantee that `pmlogger.service` will come into existence?

Reply:
I thought python3-pcp => pcp-libs => pcp is how the dependency chain worked. Let's check.

Reply:
E.g. in Debian, [pcp](https://packages.debian.org/sid/pcp) depends on [python3-pcp](https://packages.debian.org/sid/python3-pcp), but not the other way around. A mutual dependency is broken. Conceptually, the app (pcp) depends on the lib, but conceptually a Python module to interface with pcp data (e.g. archive files) mustn't pull in systemd services and such.

As `cockpit-pcp` depends on `pcp`, I think it'd make sense to keep that behaviour for now -- i.e. install `pcp` on demand, not `python3-pcp`. That even gets rid of the arch special-case.